flatflow/scheduler/profiler.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2024, The FlatFlow Authors.
```

Comment:
Fix typo: ``# Copyright 2024 The FlatFlow Authors``
I also suggest to move this file, including ``__init__.py``, to another directory named ``flatflow/torch/profiler``.